src/test/java/org/jenkinsci/plugins/envinject/EnvInjectPluginConfigurationTest.java

```diff
@@ -46,4 +47,14 @@ private void testRoundtrip(EnvInjectPluginConfiguration config) throws IOExcepti
         assertEquals("Value of enableLoadingFromMaster differs after the reload",
                 config.isEnableLoadingFromMaster(), reloaded.isEnableLoadingFromMaster());
     }
+
+    @Test
+    public void testConfigAsCode() throws Exception {
```

Comment:
please use: `@ConfiguredWithCode("configuration-as-code.yml")`
https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/PLUGINS.md#configuration-test

would be good to do a full export test, (I would move this to a different class for just testing jcasc and then use a class rule to do an import and export test)
```suggestion
    @ConfiguredWithCode("configuration-as-code.yml")
    public void testConfigAsCode() throws Exception {
```

(you'll need to add an import for ConfiguredWithCode and add the tests classifier dependency (https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/PLUGINS.md#how-to-test)

Reply:
Yeah. I was just too lazy to fix that + I did not want to modify a rule for all tests in the class.
Will kepp as is, but we can definitely refactor it later